packages/config/src/layer2s/loopring.ts

```diff
@@ -265,21 +265,21 @@ export const loopring: Layer2 = {
       }),
       discovery.getContractDetails(
         'LoopringIOExchangeOwner',
-        'Contract used by the Prover to submit exchange blocks with zkSNARK proofs that are later processed and verified by the BlockVerifier contract.',
+        'Contract used by the Prover to submit exchange blocks with zkSNARK proofs that are later processed and verified by the BlockVerifier contract. Its owner (ProxyOwner) can use this contract to give or revoke permissions to submit blocks and to open block submission to everyone.',
       ),
       discovery.getContractDetails(
         'DefaultDepositContract',
         'ERC 20 token basic deposit contract. Handles user deposits and withdrawals.',
       ),
       discovery.getContractDetails(
         'LoopringV3',
-        'Contract managing LRC staking for exchanges (One Loopring contract can manage many exchanges).',
+        'Contract managing LRC staking for exchanges (one Loopring contract can manage many exchanges). Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.',
```

Comment:
I think this belongs to Permissioned Addresses section, WYT?

Reply:
done